kots-exporter/kots-exporter.sh

```diff
@@ -354,6 +374,18 @@ output_message(){
     echo "- $path/output/helm-values.yaml"
     echo ""
     echo "-------------------------------------------------------------------------"
+
```

Comment:
you could replace most of this by adding a TTL line to the job's manifest

Reply:
@soulchips I agree, but in case of job failure
* Job will not be deleted
* User will proceed with the upgrade which we don't want

Reply:
job is deleted if fails actually. it may retry a few times based on your backoff limit but once it hits the max, it deletes itself.

i agree we want to notify the customer if the job fails however! so no issues here!